finmodel/src/render.py

```diff
@@ -74,24 +91,30 @@ def render_df(df):
     # -----------------------------------------------------------------------------------------------------------------------------------------------
     # Formatting column widths and justifications
     for column in columns:
-        if column == "YOY\nGROWTH" or column == "GIZMO\nMIX":
+        if column == "YOY\nGROWTH" or column == "GIZMO\nMIX" or column == "SALES\n(USD)":
```

Comment:
# TODO: make this dynamic or an arg in the render func call

Reply:
in other words, we should write a rendering class.